app/views/content/is-teaching-right-for-me/computing/_article.html.erb

```diff
@@ -0,0 +1,137 @@
+<div class="row inset">
+  <section class="col col-720">
+    <p>
+      Teaching computing is an important role to equip pupils with knowledge and skills that will help them succeed in a  world dependent on digital technology.</p>
+
+<p>You'll inspire pupils to access new fields and careers like quantum computing and artificial intelligence. You could inspire the next generation of computer scientists. </p>
+  </section>
+
+  <section class="col col-space-s col-content-max">
+    <%= render TextBlockWithTwoImagesComponent.new(
+      heading: "Help your pupils learn new skills",
+      colour: "pink",
+      image_1_path: "static/images/content/hero-images/0035.jpg",
+      image_1_alt: "",
+      image_2_path: "static/images/content/hero-images/0037.jpg",
+      image_2_alt: "",
+    ) do %>
+      <p>quote</p>
+<p><b>quote</p>
+    <% end %>
+  </section>
+
+  <section class="col col-720 col-horizontal col-space-m">
+    <div>
+      <h2>What you'll be teaching</h2>
+      <p>Computing for key stages 1 to 4 focuses on 3 pillars:</p>
+<ul>
+<li>information and communication technology  </li>
+
+<li>digital literacy </li>
+<li>computer science </li>
+</ul>
+<p>At GCSE and A Level you'll teach computer science. You could teach the <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-computing-programmes-of-study/national-curriculum-in-england-computing-programmes-of-study">national curriculum</a>, with opportunities to develop your students’ computational thinking.  </p>
+<p>Themes you'll likely cover for 11 to 14 year olds (key stage 3): </p>
+<p>
+When teaching the lower key stages, you could use interactive exercises during your lessons, like creating a robot car.   </p>
+      <ul class="vertical-tags">
+        <li class="tag">Algorithms</li>
+        <li class="tag">Programming and debugging   </li>
+        <li class="tag">Creative projects </li>
+        <li class="tag">Use of logical reasoning</li>
+        <li class="tag">Computer systems</li>
+        <li class="tag">Uses of technology  </li>
+        <li class="tag">Online safety </li>
+
+      </ul>
+
+      <p>Themes you'll likely cover for 14 to 16 year olds (key stage 4): </p>
+      <p>You'll teach computer science as a GCSE qualification. You'll teach pupils how to develop their capability, creativity and knowledge in computing. </p>
+      <ul class="vertical-tags">
+        <li class="tag">Computational thinking</li>
+        <li class="tag">Digital media and information technology  </li>
+        <li class="tag">Computer networks </li>
+        <li class="tag">Cyber security  </li>
+        <li class="tag">Data representation </li>
+        <li class="tag">Digital technology   </li>
+      </ul>
+      <p>For 16 to 18 year olds (key stage 5), you'll teach computer science as a A Level qualification and encourage pupils to develop a broad range of the knowledge and skills, preparing them for further learning or employment, with themes like:  </p>
```

Comment:
@caitlin-reidy think you might have meant to say their knowledge and skills not the?